spark/graphar/src/main/java/org/apache/graphar/info/EdgeInfo.java

```diff
@@ -117,13 +118,22 @@ private EdgeInfo(
         this.version = version;
     }
 
+    public static EdgeInfo load(String edgeInfoPath) throws IOException {
+        return load(edgeInfoPath, new Configuration());
+    }
+
     public static EdgeInfo load(String edgeInfoPath, Configuration conf) throws IOException {
         if (conf == null) {
             conf = new Configuration();
         }
-        Path path = new Path(edgeInfoPath);
-        FileSystem fileSystem = path.getFileSystem(conf);
-        FSDataInputStream inputStream = fileSystem.open(path);
+        return load(edgeInfoPath, FileSystem.get(conf));
+    }
+
+    public static EdgeInfo load(String edgeInfoPath, FileSystem fileSystem) throws IOException {
+        if (fileSystem == null) {
```

Comment:
I think we should raise error if the given Configuration or FileSystem is null. If we use a default `new Configuration()` and can not read the path(which failed like user give a null but path is an oss path), it can not tell that the reason of read failed is that user pass a invalid Configuration/FileSystem.

Reply:
fixed